### PR TITLE
CNV-8819 Enabling hotplug feature in Kubevirt

### PR DESCRIPTION
**Pull request by ydayagi**
Signed-off-by: Yaron Dayagi <ydayagi@redhat.com>

CNV-8819 As a user I want an easy method for enabling hotplug feature in Kubevirt

### CODE REVIEW

File: deploy/example/kubevirt-config.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  feature-gates: DataVolumes,SRIOV,LiveMigration,CPUManager,CPUNodeDiscovery,Sidecar,Snapshot,HotplugVolumes
+  selinuxLauncherType: virt_launcher.process
+  smbios: |-
+    Family: KubeVirt
+    Manufacturer: KubeVirt
+    Product: None
+kind: ConfigMap
+metadata:
+  name: kubevirt-config
+  namespace: kubevirt
```
Comment (chenyosef) [inline deploy/example/kubevirt-config.yaml:13]: Please re-order the lines according to the convention (apiVersion, than kind, metadata and only then data)

**Review by chenyosef — CHANGES_REQUESTED** [review]

File: deploy/example/kubevirt-config.yaml (hunk 1 above)
Comment (ydayagi) [reply deploy/example/kubevirt-config.yaml]: Fixed

**Pull request merged**